src/compressAlignment.hpp

```diff
@@ -192,22 +210,32 @@ class shasta::compressAlignment::Format2 {
     }
 
     Format2(
-         int32_t skip0Argument,
-         int32_t skip1Argument,
-         uint32_t nArgument)
-     {
-         SHASTA_ASSERT(skip0Argument >= -512);
-         SHASTA_ASSERT(skip0Argument <= 511);
-         SHASTA_ASSERT(skip1Argument >= -512);
-         SHASTA_ASSERT(skip1Argument <= 511);
-         SHASTA_ASSERT(nArgument <= 512);
+        int32_t skip0Argument,
+        int32_t skip1Argument,
+        uint32_t nArgument):
+        formatIdentifier(ID)
+    {
+        SHASTA_ASSERT(skip0Argument >= -512 && skip0Argument <= 511);
```

Comment:
Or use a call to `ok`?

Reply:
You mean `SHASTA_ASSERT(ok(skip0Arg, skip1Arg, nArgument));`?

Reply:
Yes.